datasets/hk/legco/crawler.py

```diff
@@ -0,0 +1,186 @@
+"""
+Crawler for extracting names and constituencies of Hong Kong
+Legislative Council members.
+"""
+
+from typing import Dict, Union, NamedTuple, Any
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+
+MEMBER_URL_FORMAT = (
+    "https://app4.legco.gov.hk/mapi/{lang}/api/LASS/getMember?member_id={member_id}"
+)
+
+
+class MemberPages(NamedTuple):
+    """Member pages in English, Traditional and Simplified Chinese"""
+
+    en: Dict[str, Any]
+    hant: Dict[str, Any]
+    hans: Dict[str, Any]
+
+
+def crawl_member_pages(context: Context, member_id: int) -> MemberPages:
+    # Retrieve member pages in English, traditional and simplified Chinese
+    pages = []
+    for lang in "en", "tc", "cn":
+        member_url = MEMBER_URL_FORMAT.format(lang=lang, member_id=member_id)
+        response = context.fetch_json(member_url, cache_days=7)
+        pages.append(response["data"])
+    return MemberPages(*pages)
+
+
+def crawl_person(
+    context: Context,
+    member: Dict[str, Union[str, int]],
+    pages: MemberPages,
+):
+    """Fetch personal information for a Legislative Council member and
+    create a Person entity."""
+    person = context.make("Person")
+    person.id = context.make_id(member.pop("member_id"), member.pop("search_key"))
+    context.log.info("Unique ID {person_id}".format(person_id=person.id))
+    # Add names in English and both Chinese writing systems
+    h.apply_name(person, full=pages.en.pop("name"), lang="en")
+    h.apply_name(person, full=pages.hant.pop("name"), lang="zh_hant")
+    h.apply_name(person, full=pages.hans.pop("name"), lang="zh_hans")
+    for email in pages.en.pop("email_address", []):
+        if email:
+            context.log.info("Email: {email}".format(email=email))
+            person.add("email", email)
+    for url in pages.en.pop("homepage", []):
+        if url:
+            context.log.info("Web: {url}".format(url=url))
+            person.add("website", url)
+    for phone in ("office_telephone", "mobile_phone"):
+        for number in pages.en.pop(phone, []):
+            if number:
+                context.log.info("Phone: {number}".format(number=number))
+                person.add("phone", number)
+    title = pages.en.pop("title")
+    if title and title != "-":
+        context.log.info("Title: {title}".format(title=title))
+        person.add("title", title)
+    for qual in pages.en.pop("qualification", []):
+        if qual:
+            context.log.info("Education: {qual}".format(qual=qual))
+            person.add("education", qual)
+    for address in pages.en.pop("office_address", []):
+        if address:
+            context.log.info("Address: {address}".format(address=address))
+            address_entity = h.make_address(context, address)
+            h.apply_address(context, person, address_entity)
```

Comment:
We don't normally add education and addresses for PEPs, although now that it's done, it doesn't cause harm. 
I've made a start to documenting which fields are priority and which aren't in https://zavod.opensanctions.org/priorities/ but I realise nothing's linking to it yet so I can't expect anyone to find it :)

Reply:
OK, I can leave them in - the same data appears to be in Wikidata as well.